Exercise 4/check-prime.js

```diff
@@ -0,0 +1,12 @@
+function checkPrime(number) {
+  let isPrime = false;
+  for (let initial = 2; initial < number; initial++) {
```

Comment:
As a suggestion, it is okay to use already established conventions like using the `i` when looping/